client/state/test/index.js

```diff
@@ -0,0 +1,48 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		createReduxStore = require( 'state' ).createReduxStore;
```

Comment:
Oh right, the works too. I think I originally saw that stack trace and then started to mock out the analytics middleware. It feels like require at the top is probably the cleanest option.